pkg/security/controller/namespace_scc_allocation_controller.go

```diff
@@ -29,7 +29,8 @@ import (
 	coreapi "k8s.io/kubernetes/pkg/apis/core"
 
 	securityv1 "github.com/openshift/api/security/v1"
-	securityv1client "github.com/openshift/client-go/security/clientset/versioned/typed/security/v1"
+	securityinternalv1 "github.com/openshift/api/securityinternal/v1"
+	securityv1client "github.com/openshift/client-go/securityinternal/clientset/versioned/typed/securityinternal/v1"
```

Comment:
nit: since you're using both security and securityinternal, I'd prefer this have securityinternalv1client - long but clear it's the internal